### DISCUSSION TITLE
Can I use the all-contig-anonotation file to reannotate γδTCR? Even though most of the files are low confident in Cell Ranger (v7.0) all-contig-anonotation output

### DISCUSSION THREAD

**Discussion by HACKYANG123** [tuonglab/dandelion#322] [Q&A]
Dear Kelvin,

I read your article and found Dandelion really a powerful tool.

I want to detect T cells with gamma-delta chains in my V(D)J data. I used Cell Ranger (v7.0) to process data of TCR libraries enriched for TRG/D chains. But most of the contig were filtered out because is_cell false and high_confidence false. Below are the biggest reasons for why the barcodes were filtered out.

 593 "not_enough_junction_support",

309049 "not_enough_reads_per_umi",

309690 "not_enough_umis_tcr_or_denovo"

 

So I want to reannotate with Dandelion to get more contig of γδTCR. Can I use the all-contig-anonotation file for analysis? Even though most of the files are low confident in Cell Ranger (v7.0) all-contig-anonotation output ?
![image](https://github.com/zktuong/dandelion/assets/64158443/4239adae-60f5-42b1-9814-16abb4a1de06)

I have attached the files of all_contig_annotations.csv and filter_contig_annotations.csv.

Thank you so much!

Best,

Yan

**Comment by zktuong** [comment]
hi @HACKYANG123, i think you can try for sure - cellranger will still return their reconstructed sequence that dandelion can annotate with igblast so maybe that might help abit?

you can provide `all_contig_annotations.csv` and `all_contig.fasta` to Dandelion to do the processing.

dandelion would put more priority on the cell barcodes that survive the standard QCs for the gene expression side.
